pkg/api/api.go

```diff
@@ -14,41 +17,85 @@ import (
 	"github.com/fossology/LicenseDb/pkg/auth"
 	"github.com/fossology/LicenseDb/pkg/db"
 	"github.com/fossology/LicenseDb/pkg/models"
+	"github.com/fossology/LicenseDb/pkg/utils"
 	"github.com/gin-gonic/gin"
+	"github.com/swaggo/files"
+	"github.com/swaggo/gin-swagger"
```

Comment:
```suggestion
	swaggerFiles "github.com/swaggo/files"
	ginSwagger "github.com/swaggo/gin-swagger"
```
It works without the label, probably because there is a custom alias set somewhere, static check tools will remove the imports if the label is not used in the code. So making it explicit would be better.

Reply:
Done.